docs/learn/documentation/versioned/hadoop/overview.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: page
+title: Batch Processing Overview
+---
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+
+Samza provides a unified data processing model for both stream and batch processing. In Samza batch is treated as a special case of streaming: the sources are bounded data sets in batch, whereas unbounded data sets in streaming. Under the hood, it’s the same highly-efficient stream-processing engine.
+
+![diagram-medium](/img/{{site.version}}/learn/documentation/hadoop/unified_batch_streaming.png)
```

Comment:
I can't comment on the image directly, so commenting here:
Prefer consistency in the descriptions of the inputs (plurality and set vs source). So...
Unbounded data source
Bounded data source